source/projects/private-voice-assistant/voice_remote_classic_phone.markdown

```diff
@@ -0,0 +1,154 @@
+   * Say a [supported voice command](/docs/assist/builtin_sentences/). For example, *Turn off the light in the kitchen*.
+   * You can also ask a question, such as
+     *  *Is the front door locked?*
+     *  *Which lights are on in the living room?*.
+
+
+## Troubleshooting
+
+Things are not working as expected? This section lists a few steps that may help you troubleshoot issues.
+
+### Troubleshoot Grandstream
+
+If you’re unable to call Home Assistant, confirm the following settings in your Grandstream device’s web interface.
+
+1. On the **FXS Port** tab, check the **Preferred Vocoder** list. 
+   * Make sure that **OPUS** is selected for one of the choices:
+   ![Vocoder OPUS option](/images/assist/grandstream_vocoder.png) 
+1. Under **OPUS Payload type**, make sure the value is `123`. It's the default option.
+   ![Vocoder OPUS payload type](/images/assist/grandstream_opus_payload.png) 
+
+
+### Test if the phrase works without voice
+
+Use this procedure to test if the phrase itself works, without using the voice.
+
+1. Go to the **Developer Tools** and open the **Services** tab.
+1. Under **Service**, select **Conversation: Process**.
+1. If you are using another language than the one defined under your user settings:
+   * Select the **Language** checkbox and enter the language code. 
+1. Select the **Text** checkbox and enter the phrase you are using.
+1. Select **Call service**.
+![Call a conversation process from the services tab](/images/assist/conversation-service-01.png)
+   * If the phrase does not work, try a variant. For example, if *Turn on the lights* doesn't work, try: *Turn on the lights in the kitchen*.
+   * Check if your phrase is [supported](/docs/assist/builtin_sentences/).
+
+### View debug information
+
+1. Go to **Settings** > **Voice assistants**.
+1. From the list of assistants, select your assistant.
+1. In the dialog, select **Debug**.
+1. At the top of the screen, from the dropdown menu, select the run you are interested in.
+![Debug speech-to-text](/images/assist/assistant-debug-02.png)
+
+### View the logs
+
+1. Go to {% my logs title="**Settings** > **System** > **Logs**" %}.
+![View system logs](/images/assist/assistant-view-system-logs-01.png)
+   * Click on the log entry to view a more detailed description.
+
+## About the analog phone
+
+You can use any analog landline phone, ideally one that already has an RJ11 socket.
+During creation of this tutorial this classic phone from 1953 was used:
+[*Tischstation Mod.29 HF-TR* by Autophon AG](https://www.radiomuseum.org/r/autophon_tischstation_mod29_hf_tr.html).
+![Analog phone Tischstation Mod.29 by Autophon AG](/images/assist/autophon-mod-29.jpg)
+
+## FAQ
+
+**Which languages are supported?**
+[These](https://developers.home-assistant.io/docs/voice/intent-recognition/supported-languages/) are the languages supported by Home Assistant.
```

Comment:
I think just linking `this` or `these` words in a sentence is generally a bit unclear in intention. Especially in this case, we can add a little more context to the link by linking the sentence.

```suggestion
[These are the languages supported by Home Assistant](https://developers.home-assistant.io/docs/voice/intent-recognition/supported-languages/).
```